specification/marketplacecatalog/data-plane/Products/preview/2023-01-01-preview/products.json

```diff
@@ -7,6 +7,11 @@
     "termsOfService": "None",
     "version": "2023-01-01-preview"
   },
+  "x-typespec-generated": [
+    {
+      "emitter": "@azure-tools/typespec-autorest"
+    }
+  ],
```

Comment:
where is your TypeSpec?

Reply:
Hi @weidongxu-microsoft,
Apologies for the confusion, this was just my experimentation, have reverted the this change